pkg/scheduler/utilities_test.go

```diff
@@ -376,9 +587,77 @@ func getTestUserGroup() security.UserGroup {
 }
 
 func assertUserGroupResource(t *testing.T, userGroup security.UserGroup, expected *resources.Resource) {
-	ugm := ugm.GetUserManager()
-	userResource := ugm.GetUserResources(userGroup)
-	groupResource := ugm.GetGroupResources(userGroup.Groups[0])
+	expectedQueuesMaxLimits := make(map[string]map[string]interface{})
+	expectedQueuesMaxLimits["root"] = make(map[string]interface{})
+	expectedQueuesMaxLimits["root.default"] = make(map[string]interface{})
+	expectedQueuesMaxLimits["root"][maxresources] = resources.NewResourceFromMap(map[string]resources.Quantity{"memory": 10, "vcores": 10})
+	expectedQueuesMaxLimits["root.default"][maxresources] = resources.NewResourceFromMap(map[string]resources.Quantity{"memory": 5, "vcores": 5})
+	expectedQueuesMaxLimits["root"][maxapplications] = uint64(2)
+	expectedQueuesMaxLimits["root.default"][maxapplications] = uint64(1)
+	internalAssertUserGroupResource(t, userGroup, expected, expectedQueuesMaxLimits)
+}
+
+func internalAssertUserGroupResource(t *testing.T, userGroup security.UserGroup, expected *resources.Resource, expectedQueuesMaxLimits map[string]map[string]interface{}) {
```

Comment:
I'd rename this to `assertUserGroupResourceMaxLimits()`, or maybe even just `assertResourcesMaxLimits()` (and the existing can be changed to `assertResources()`). Otherwise I have to scan with my eyes to see why it's called "internal".